janus_server/src/aggregator/aggregation_job_creator.rs

```diff
@@ -279,6 +281,101 @@ impl<C: Clock + 'static> AggregationJobCreator<C> {
             })
             .await?)
     }
+
+    /// Look for combinations of client reports and collect job aggregation parameters that do not
+    /// yet have a report aggregation, and batch them into new aggregation jobs. This should only
+    /// be used with VDAFs that have non-unit type aggregation parameters.
+    // This is only used in tests thus far.
+    #[cfg(test)]
+    #[tracing::instrument(skip(self), err)]
+    async fn create_aggregation_jobs_for_task_with_param<const L: usize, A: vdaf::Aggregator<L>>(
+        &self,
+        task: &Task,
+    ) -> anyhow::Result<()>
+    where
+        for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
```

Comment:
nit:
```suggestion
    async fn create_aggregation_jobs_for_task_with_param<const L: usize, A>(
        &self,
        task: &Task,
    ) -> anyhow::Result<()>
    where
        A: vdaf::Aggregator<L>,
        for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
```

Having said that, what you have is consistent with some other methods in the file, so clearly we don't have an established or enforced style here. Meaning you should do whatever you like best.